src/index.js

```diff
@@ -1,17 +1,33 @@
-import {plugin} from 'postcss';
+import {plugin as postCSSPlugin} from 'postcss';
```

Comment:
In this case it might be nicer to load `postcss`;

```js
import postcss from 'postcss';

// ...

export default postcss.plugin('postcss-devtools', opts => {
```